csharp/rtl/Constants.cs

```diff
@@ -61,7 +61,7 @@ public struct Constants
 
         public const string DefaultApiVersion = "4.0";
         public const string AgentPrefix = "CS-SDK";
-        public const string LookerVersion = "23.18";
+        public const string LookerVersion = "23.17";
```

Comment:
This isn't based on a CI image then? Should it be 23.16 still?

Reply:
No, somehow the version number got screwed up last time. But when we publish this again they'll be fixed.